### ISSUE TITLE
Need help to get OpenY Drupal distribution in DigitalOcean marketplace

### ISSUE THREAD

**Issue by podarok** [digitalocean/marketplace-partners#59]
Hi, dear team
We'd like to create our own marketplace app for the customers to start with.
It was not easy to find how this could be done.
From what I already understood we should create a droplet. make a snapshot from it and then somehow to send for approval.
The last steps is not clear. Looks like the only way is to ask. 
Could you provide me a details of how to push our distro to DigitalOcean MarketPlace?

Details
github repo https://github.com/ymcatwincities/openy
Sandbox http://sandbox.openymca.org

Basically sandbox is a thing that will be automatically installed on DigitalOcean Droplet to be used in MarketPlace. We'll take care about all underlying cleanups and scripts.

**Comment by jgannondo** [comment]
@podarok mind emailing us on one-clicks-team@digitalocean.com? The submission process is currently in beta so we'd like to work with you directly on this. Thanks!
